docs/Reference/Console/Features/ocr.md

```diff
@@ -0,0 +1,87 @@
+--8<-- "References/abbreviations.md"
+
+Intel® One-Click Recovery (OCR) allows IT administrators to remotely trigger a secure and reliable boot to a recovery application, ensuring recovery from system failures, bare-metal scenarios, or connectivity issues using Intel AMT's out-of-band (OOB) connection.
+
+Console now supports triggering HTTPS Boot, allowing IT administrators to initiate recovery via a secure network boot from a specified URL.
+
+### Supported Recovery Options
+
+OCR supports three recovery modes:
+
+- **UEFI HTTPS Network Boot**: Perform a network-based recovery using encrypted HTTPS.
+
+- **Microsoft Windows Recovery Environment (WinRE)**: Boot into Windows Recovery for troubleshooting and repair.
+
+- **Local Pre-Boot Application (PBA)**: Launch a locally installed recovery or diagnostic tool.
+
+!!! note "Note - OCR support"
+
+    Console currently supports HTTPS Network Boot over a wired connection. Additional recovery options will be added in future releases.
+
+### Prerequisites for HTTPS Boot
+
+Before using HTTPS Boot OCR, ensure the following prerequisites are met:
+
+1. Only Intel AMT version 15 and later support One Click Recovery feature.
+2. Enable HTTP/HTTPS Boot in BIOS settings. This may be disabled by default on some devices.
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_HTTPSBOOT_BIOS.png" alt="Figure 1: Enable HTTPS Boot in BIOS">
+    </figure>
+3. Set up an HTTPS server to host the ISO. For faster booting, run it on the same network as the device.
+    - This guide references using [simple-webserver](https://github.com/jsastriawan/simple-webserver) for the HTTPS server setup. 
+    - In this example, the HTTPS server is running on the same machine as the Console: https://192.168.88.250:5443/ubuntu.iso.
+    - The hosted ISO is a full Ubuntu LTS image served via simple-webserver.
+
+### Triggering OCR via Console
+
+1. Connect to the Intel AMT device using Console over TLS.
```

Comment:
Does it have to be TLS?

Reply:
Reading through this, you have in the troubleshooting that this is required. I would add an "IMPORTANT" Admonition box here that this must be the case, and non tls will not work.